taxonomies/ingredients.txt

```diff
@@ -86318,7 +86318,7 @@ wikipedia:en:https://en.wikipedia.org/wiki/Xylose
 # Tonka beans are prohibited for use in food in the US: https://www.govinfo.gov/content/pkg/CFR-2006-title21-vol3/xml/CFR-2006-title21-vol3-sec189-130.xml
 en:tonka bean
 es:haba tonka
-de:Tonkabohn, Tonkabohnen
+de:Tonkabohn, Tonkabohnen, Tonkabohne
```

Comment:
I think "Tonkabohn" is probably a typo. If you search it on the internet, it only results in a handful of entries, which seems to confirm this. Let's just remove that alias, IMHO.
```suggestion
de:Tonkabohne, Tonkabohnen
```

Reply:
You are right, I modified it, thanks